video/out/opengl/ra.h

```diff
@@ -182,6 +182,7 @@ struct ra_renderpass_input {
     int dim_v;              // vector dimension (1 for non-vector and non-matrix)
     int dim_m;              // additional matrix dimension (dim_v x dim_m)
     // Vertex data: byte offset of the attribute into the vertex struct
+    size_t offset;
```

Comment:
I don't really understand why you don't like this (it's just part of weirdly mashing together vertex attribute description with other stuff, but everyone seems to do this, i.e. Vulkan using texture formats to describe entries). But OK.

Reply:
I guess it's the type confusion that bothers me: offsets are units of bytes (i.e. size_t), bindings are just generic natural numbers. The bindings are counted 0, 1, 2, 3 etc.; the offsets depend on the size of the data.

Reply:
More than enough code in mpv and ffmpeg uses `int` for byte sizes too.